### PR TITLE
Update 0.3.1

### PR DESCRIPTION
**Pull request by Jrice1317**
### Changes made
- Initialized Feedstock from conda-forge
- Added jinja variables
- Added dev & doc urls, license family & description
- Reformatted according to recipe standards

1. Jira Ticket: [PKG-856](https://anaconda.atlassian.net/browse/PKG-856)
2. Upstream: https://github.com/agoose77/hatch-nodejs-version/tree/master
3. Change log: https://github.com/agoose77/hatch-nodejs-version/compare/v0.3.0...v0.3.1

### CODE REVIEW

**Comment by Jrice1317** [comment]
> Does `wheel` still need to be present if `hatchling` exists in the `host` section?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,9 +42,13 @@ test:
 
 about:
   home: https://github.com/agoose77/hatch-nodejs-version
-  summary: Hatch plugin for versioning from a package.json file
+  dev_url: https://github.com/agoose77/hatch-nodejs-version/tree/master
```
Comment (cbouss) [inline recipe/meta.yaml]: Can you make the dev_url more generic?
https://github.com/agoose77/hatch-nodejs-version/

**Comment by cbouss** [comment]
> > Does `wheel` still need to be present if `hatchling` exists in the `host` section?

I am not sure but in doubt I would still add it.

**Review by varlackc — APPROVED** [review]
Other than the issue with the `dev_url` everything else seems in order.

File: recipe/meta.yaml (hunk 1 above)
Comment (Jrice1317) [reply recipe/meta.yaml]: Done!

**Review by cbouss — APPROVED** [review]

**Pull request merged**